src/modules/storage/TransactionsStorage.ts

```diff
@@ -0,0 +1,162 @@
+/*******************************************************************************
+
+    The class that insert and read the transactions into the database.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import * as sqlite from 'sqlite3';
+import { Storages } from './Storages';
+
+/**
+ * The class that insert and read the transactions into the database.
+ */
+export class TransactionsStorage extends Storages
```

Comment:
Why do we need to have those `Storages` class ? We have a single DB where we store everything. Why not just a single `Database` class that does the work ? We don't put single transactions at the moment.